HW_3/task_3.js

```diff
@@ -0,0 +1,68 @@
+/*
+1*. Создать программу, которая будет принимать на вход СЛОВО (создать переменную со словом), 
+  и выводить в консоль количество гласных и согласных букв в этом слове. 
+  Ответ должен выводиться шаблонным литералом вида word contains x vowels and y consonants
+*/
+
+let word = 'WORD'.toLocaleLowerCase()
+vowels = "aeiou"
+vowels_count = 0
+consonants_count = 0
+for(let i of word){
+    if(vowels.includes(i)){
+        vowels_count++;
+        }
+   else {
+    consonants_count++;
+        }
+}
+console.log(`vowels: ${vowels_count}, word contains: ${consonants_count}`);

```

Comment:
Test your solution with this case `ab21`
You should get `vowels: 1, consonants: 1`